helix-view/src/editor.rs

```diff
@@ -19,6 +19,19 @@ pub use helix_core::register::Registers;
 use helix_core::syntax;
 use helix_core::Position;
 
+/// File path with row and column.
```

Comment:
I don't completely understand everything that's going on here, and this is my first time seriously reviewing a PR, but is there a reason that `File` can't be baked into `Document`? Or is this to make the intent of the code clearer?

Reply:
Ah, this file is not the normal file, this is the file argument being passed into `:open` or `hx <file>`.